doc/vaulted-console.1.md

```diff
@@ -0,0 +1,44 @@
+vaulted-console 1
+=============
+
+NAME
+----
+
+vaulted console - Opens the AWS console in the default web browser
+
+SYNOPSIS
+--------
+
+`vaulted console`  
+`vaulted console` *name* [*OPTIONS*]  
+`vaulted console --assume` *arn* [*OPTIONS*]  
+`vaulted console --duration` *duration* [*OPTIONS*]
+
+DESCRIPTION
+-----------
+
+Opens the AWS console in the default web browser. Uses either the credentials in the current environment or the credentials in the specified vault. Console sessions either use the passed in duration, the provided vault's duration, or defaults to 1 hour.
```

Comment:
I feel like this description can use a little work.  For starters, "uses either the credentials in the current environment or the credentials in the specified vault" doesn't really capture the order of preference.

Second, "Console sessions use" seems unnecessary, given that this is a document on the console command, so the duration is implied to correspond to the console.  Additionally, it feels a bit disjointed in how we are going about describing the methods by which a duration is supplied, made worse when we seem to take on the duration range after the rest.  

Perhaps we should take a pass on this and see if we can improve the wording a bit?

Reply:
done